### ISSUE TITLE
check "extras" not just "deps" ?

### ISSUE THREAD

**Issue by bjarthur** [JuliaRegistries/CompatHelper.jl#166]
would it make sense to check everything that's in "compat", not just the subset that's also listed in "deps"?  the stuff in "extras" seems not to be checked.  see https://github.com/GiovineItalia/Compose.jl/issues/374

**Comment by exaexa** [comment]
Hello, is there anything I can bump/help with to get this implemented?

**Comment by sethaxen** [comment]
I think with #458, a single-line change to loop over `["deps", "weakdeps", "extras"]` instead of just `["deps", "weakdeps"]` would resolve this issue.

**Comment by exaexa** [comment]
yeah looks like that's it. I'll try to open another PR when #458 gets merged. (Not sure if adding more stuff to that PR would be good)
